### PR TITLE
Feature/add layout style

### PR DESCRIPTION
**Pull request by tsuifei**
- Confirm TailwindCss run normally.
- Adding layout style
- Adding RWD menu

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,30 +1,64 @@
...
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M4 6h16M4 12h16M4 18h16"
+            ></path>
+          </svg>
+        </button>
+
+
+        {/* <div className="hidden w-full md:block md:w-auto" id="navbar-default">
```
Comment (codeewander) [inline src/components/Header.jsx]: remove unused code

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,30 +1,64 @@
...
+      <nav >
+      <div className="max-w-7xl mx-auto flex items-center justify-between h-16 px-4 border-b border-solid border-slate-600">
+        <Link to="/" className="flex items-center">
+            <img src="../../public/logo.svg" className="h-12 mr-3" alt="Chemins de Compostelle Logo" />
+            <span className="self-center text-base font-semibold whitespace-nowrap lg:text-2xl">Chemins de Compostelle</span>
+        </Link>
+        <div className="hidden md:block">
+          <Menu />
+        </div>
+        <button
```
Comment (codeewander) [inline src/components/Header.jsx:18]: change the hamburger button color to "green"

File: src/App.jsx [hunk 3]
```diff
@@ -16,8 +16,7 @@ import NotFound from "./pages/NotFound";
 
 function App() {
   return (
-    <div className="App">
-      <h2 className="bg-gray-800">Les chemins de compostelle</h2>
+    <div className="App max-w-7xl mx-auto p-2 mb-40">
```
Comment (codeewander) [inline src/App.jsx]:
```suggestion
    <div className="max-w-7xl mx-auto p-2 mb-40"> 
```

Comment (codeewander) [reply src/App.jsx]: class App is not exist

File: src/components/Header.jsx [hunk 4]
```diff
@@ -1,30 +1,64 @@
...
+            </li>
+            <li>
+              <Link to='/qa' className="block py-2 pl-3 pr-4 text-gray-900 rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-green-700 md:p-0" >Q&A</Link>
+            </li>
+          </ul>
+        </div> */}
+      </div>
+      <div className="md:hidden">
+        {showMobileMenu && <Menu />}
+      </div>
```
Comment (codeewander) [inline src/components/Header.jsx:38]:
```suggestion
        {showMobileMenu && <Menu />}
```

Comment (codeewander) [reply src/components/Header.jsx:38]: no need to add div here, it is redundant

File: src/components/Menu.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+const RwdMenu = () => {
```
Comment (codeewander) [inline src/components/Menu.jsx]: Nit: component name should be the same as file name

File: tailwind.config.js [hunk 6]
```diff
@@ -1,10 +1,18 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    "./index.html",
+    // "./index.html",
```
Comment (codeewander) [inline tailwind.config.js:4]: remove comment

File: tailwind.config.js [hunk 7]
```diff
@@ -1,10 +1,18 @@
...
   ],
   theme: {
+    // stroke: {
+    //   current: 'currentColor',
+    // },
+    // stroke: theme => ({
+    //   'red': theme('colors.red.500'),
+    //   'green': theme('colors.green.500'),
+    //   'blue': theme('colors.blue.500'),
+    // }),
```
Comment (codeewander) [inline tailwind.config.js]:
```suggestion
    stroke: {
       current: 'currentColor',
     },
     stroke: theme => ({
       'red': theme('colors.red.500'),
       'green': theme('colors.green.500'),
       'blue': theme('colors.blue.500'),
     }),
```

File: tailwind.config.js (hunk 6 above)
Comment (codeewander) [inline tailwind.config.js:4]:
```suggestion
     "./index.html",
```

File: src/components/Header.jsx (hunk 4 above)
Comment (tsuifei) [reply src/components/Header.jsx:38]: This is required, if you delete the div and style, the mobileMenu will reappear in RWD lg mode. 
Or is there another way to add the style to the JSX syntax? :-)

File: src/components/Header.jsx (hunk 1 above)
Comment (tsuifei) [reply src/components/Header.jsx]: fix it commit : 902e2b

File: src/components/Header.jsx (hunk 2 above)
Comment (tsuifei) [reply src/components/Header.jsx:18]: fix it commit : 902e2b

File: src/App.jsx (hunk 3 above)
Comment (tsuifei) [reply src/App.jsx]: fix it commit : 902e2b

File: src/components/Menu.jsx (hunk 5 above)
Comment (tsuifei) [reply src/components/Menu.jsx]: fix it commit : 902e2b

File: tailwind.config.js (hunk 7 above)
Comment (tsuifei) [reply tailwind.config.js]: fix it commit : 902e2b

File: tailwind.config.js (hunk 6 above)
Comment (tsuifei) [reply tailwind.config.js:4]: fix it commit : 902e2b

**Review by codeewander — APPROVED** [review]

**Pull request merged**